interface/src/app/analyze/analysis/analysis.less

```diff
@@ -1,3 +1,37 @@
+/*
+ * styles to make sampleBin editing controls more user-friendly
+ */
+.row-control {
+  float: none;
+  .opacity(.2);
+
+  &:hover,
+  &:focus {
+    text-decoration: none;
+    cursor: pointer;
+    .opacity(.5);
+  }
+
+  button& {
+    padding: 0;
+    cursor: pointer;
+    background: transparent;
+    border: 0;
+    -webkit-appearance: none;
+  }
+}
+// allow us to use controls to help align headers, but don't show them
+th .row-control, thead .row-control {
+  visibility: hidden;
+}
+
+// sample group highlighting
```

Comment:
That's a good question. I know in some editors you can see the color by selecting the hex RGB code `#b7baff` and using a menu command so it may not be necessary. Now that you mention it, however, it is not entirely clear from this context that these two variables are colors. I will add a comment that helps with that.

- [ ] clarify that "sample group highlighting" is the *colors* used for labeling samples groups

The reason I didn't try to come up with a name that describes the colors (e.g. "purple-highlight") is that I wasn't sure we would stick with these exact colors. I wanted the variable name to describe what it is *for* more than what it *is* so we can experiment with different colors if we choose. Does that rationale make sense?